File: solution/1_Palindroom/1_Palindroom/PalindroomTest.cs
using System;
using System.Text;
using System.Collections.Generic;
using System.Linq;
using Microsoft.VisualStudio.TestTools.UnitTesting;
using palindroom;

namespace test
{
    [TestClass]
    public class PalindroomTest
    {
        [TestMethod]
        public void legeStringIsEenPalindroom() {
		    Assert.IsTrue(new Palindroom().isPalindroom(""));
	    }
	
	    [TestMethod]
	    public void eenKarakterStringIsEenPalindroom() {
            Assert.IsTrue(new Palindroom().isPalindroom("a"));
	    }
	
	    [TestMethod]
	    public void palindroom() {
            Assert.IsTrue(new Palindroom().isPalindroom("koortsmeetsysteemstrook"));
	    }

        [TestMethod]
	    public void geenPalindroom() {
            Assert.IsFalse(new Palindroom().isPalindroom("koekoeksklok"));
	    }
    }
}

File: workspace/1_Palindroom/test/palindroom/PalindroomTest.java
package palindroom;

import static junit.framework.Assert.assertFalse;
import static junit.framework.Assert.assertTrue;

import org.junit.Test;


public class PalindroomTest {
	
	@Test
	public void legeStringIsEenPalindroom() {
		assertTrue(new Palindroom().isPalindroom(""));
	}
	
	@Test
	public void eenKarakterStringIsEenPalindroom() {
		assertTrue(new Palindroom().isPalindroom("a"));
	}
	
	@Test
	public void palindroom() {
		assertTrue(new Palindroom().isPalindroom("koortsmeetsysteemstrook"));
	}
	
	@Test
	public void geenPalindroom() {
		assertFalse(new Palindroom().isPalindroom("koekoeksklok"));
	}
	

}
